user/zrepl-dsh2dsh/template.py

```diff
@@ -0,0 +1,24 @@
+pkgname = "zrepl-dsh2dsh"
+pkgver = "0.9.2"
+pkgrel = 0
+build_style = "go"
+hostmakedepends = ["go"]
+depends = ["zfs"]
+provides = [self.with_pkgver("zrepl")]
```

Comment:
drop this

Reply:
Corresponds with the comment above. I may actually provide template for both zrepl and this new fork. Than this would be needed.